NKit/Settings/RecoveryData.cs

```diff
@@ -96,7 +96,7 @@ internal RecoveryData(Settings settings, ILog log, bool isGameCube, string id8)
 
         public static string GetUpdatePartition(Settings settings, uint crc)
         {
-            Regex m = new Regex(@"\\([A-Z0-9]{40})_([A-Z]+)_" + crc.ToString("X8") + "$", RegexOptions.IgnoreCase);
+            Regex m = new Regex(@"[/|\\]([A-Z0-9]{40})_([A-Z]+)_" + crc.ToString("X8") + "$", RegexOptions.IgnoreCase);
```

Comment:
It might be wiser to use the Path.DirectorySeparatorChar character provided by the IO library. This will make sure that the regex is platform agnostic. I do not have visual studio installed at the moment but I think the code would look something like this
```suggestion
            Regex m = new Regex(Path.DirectorySeparatorChar+@"([A-Z0-9]{40})_([A-Z]+)_" + crc.ToString("X8") + "$", RegexOptions.IgnoreCase);
```
Nonetheless, I am a bit skeptical on whether the Path.DirectorySeparatorChar would be properly parsed by the Regex builder if the Path.DirectorySeparatorChar is "\"
We could propose a switch statement that properly builds the regexp.

The reason I want to avoid using [/|\\] is because in extreme cases it could match wrongly